front/src/pages/nft/[id].tsx

```diff
@@ -1,12 +1,36 @@
-import { useRouter } from "next/router";
 import React from "react";
+import { useRouter } from "next/router";
+import {useGetNFTById} from "../../services/nftService";
+import {GCImage} from "../../gc-components";
+import {NFTCard} from "../../schema/NFTCatalogSchema";
+import {GCGrid} from "../../gc-components/GCGrid";
+import {GCGridCol} from "../../gc-components/GCGridCol";
+import {useStyles} from "./SingleNFTPageStyles";
+
+type SingleNFTPageProps = {
+    data: NFTCard
+};
+
+const SingleNFTPage = (props: SingleNFTPageProps) => {
+    const id: number = Number(useRouter().query.id);
```

Comment:
id should be thrown from props maybe?

Reply:
props here will be placeholder for future possible needs, and it will be empty for now
id should be taken from url, because this is another page, we cant send it by props
or u asking something else here?

Reply:
nope